pages/about-us/newsletter/2016/02/de.md

```diff
@@ -0,0 +1,26 @@
+@title = "Februar"
+
+Einführung von share.riseup.net!
+--------------------------------------
+
+Wir würden uns freuen, wenn ihr https://share.riseup.net testet. Du kennst sicher auch das Problem, dass Anhänge beim Senden von Mails zu groß sind, oder alle sagen "Warum hast du das getan?", oder noch schlimmer: du bekommst eine fette Mail, die ich über dein Speicherlimit bringt. Dafür gibt es jetzt https://share.riseup.net, indem dort temporär große Dateien mit Wegwerflinks gespeichert werden können.
+
+https://share.riseup.net ist eine einfach nutzbarer “pastebin” und “imagebin” mit Verschlüsslung auf Anwendungsseite, gehosted von Riseup. Wenn du eine Datei teilst, wird sie verschlüsselt, bevor sie deinen Computer verlässt. Wir wissen also nicht, was du dort hochlädst.
+
+Diese Anwendungs-seitige Verschlüsselung, bei der das Program von Server herunter geladen wird, ist leicht zu umgehen, wenn der Anbieter entscheidet, eine Hintertür einzubauen, aber es ist schwer, dies zu verifzieren. Mit anderen Worten ist browserbasierte Verschlüsselung nützlich für den Dienstanbieter, Riseup in diesem Fall, weil wir keine Geheimnisse speichern, aber es garantiert nicht eure Konfidal. Für richtige Sicherheit braucht eine Anwendung, die nicht im Browser läuft.
```

Comment:
Diese Anwendungs-seitige Verschlüsselung, bei der das Program von Server herunter geladen wird, ist leicht zu umgehen, wenn der Anbieter entscheidet, eine Hintertür einzubauen. Wir haben keine Hintertür, aber es ist schwer, dies zu überprüfen. Mit anderen Worten ist browserbasierte Verschlüsselung nützlich für den Dienstanbieter, Riseup in diesem Fall, weil wir keine Geheimnisse für Euch speichern müssen. Aber es garantiert nicht Euch nicht die Geheimhaltung Eurer Daten. Für richtige Sicherheit braucht es eine Anwendung, die nicht im Browser läuft.